lib/augmentConfig.js

```diff
@@ -354,7 +361,8 @@ function applyOverrides(fullConfig, rootConfigDir, filePath) {
 	let { overrides, ...config } = fullConfig;
 
 	if (!overrides) {
-		return config;
+		// Do mergeConfigs to remove duplicate items
+		return mergeConfigs(config, {});
```

Comment:
opinion

that's overkill
i.e. if someone has only one config, which has duplicates, he ought to fix them himself